app/src/main/java/np/com/aawaz/csitentrance/Utils/Encryption.java

```diff
@@ -0,0 +1,218 @@
+    }
+
+    public String filter(String text, boolean isFromSecret) {
+        if (text.length() < 16) {
+            int remSize = 16 - text.length();
+            if (isFromSecret) return text + demo_random1.substring(0, remSize);
+            else return text + demo_random2.substring(0, remSize);
+        } else {
+            return text.substring(0, 16);
+        }
+
+    }
+
+    public byte[] encrypt(String text) throws Exception {
+        if (text == null || text.length() == 0)
+            throw new Exception("Empty string");
+
+        byte[] encrypted = null;
+
+        try {
+            cipher.init(Cipher.ENCRYPT_MODE, keyspec, ivspec);
+
+            encrypted = cipher.doFinal(padString(text).getBytes());
+        } catch (Exception e) {
+            throw new Exception("[encrypt] " + e.getMessage());
+        }
+
+        return encrypted;
+    }
+
+    public byte[] decrypt(String code) throws Exception {
+        if (code == null || code.length() == 0)
+            throw new Exception("Empty string");
+
+        byte[] decrypted = null;
+
+        try {
+            cipher.init(Cipher.DECRYPT_MODE, keyspec, ivspec);
+
+            decrypted = cipher.doFinal(hexToBytes(code));
+            //Remove trailing zeroes
+            if (decrypted.length > 0) {
+                int trim = 0;
+                for (int i = decrypted.length - 1; i >= 0; i--) if (decrypted[i] == 0) trim++;
+
+                if (trim > 0) {
+                    byte[] newArray = new byte[decrypted.length - trim];
+                    System.arraycopy(decrypted, 0, newArray, 0, decrypted.length - trim);
+                    decrypted = newArray;
+                }
+            }
+        } catch (Exception e) {
+            throw new Exception("[decrypt] " + e.getMessage());
+        }
+        return decrypted;
+    }
+
+
+    public static String bytesToHex(byte[] buf) {
+        char[] chars = new char[2 * buf.length];
+        for (int i = 0; i < buf.length; ++i) {
+            chars[2 * i] = HEX_CHARS[(buf[i] & 0xF0) >>> 4];
+            chars[2 * i + 1] = HEX_CHARS[buf[i] & 0x0F];
+        }
+        return new String(chars);
+    }
+
+
+    public static byte[] hexToBytes(String str) {
+        if (str == null) {
+            return null;
+        } else if (str.length() < 2) {
+            return null;
+        } else {
+            int len = str.length() / 2;
+            byte[] buffer = new byte[len];
+            for (int i = 0; i < len; i++) {
+                buffer[i] = (byte) Integer.parseInt(str.substring(i * 2, i * 2 + 2), 16);
+            }
+            return buffer;
+        }
+    }
+
+
+    private static String padString(String source) {
+        char paddingChar = 0;
+        int size = 16;
+        int x = source.length() % size;
+        int padLength = size - x;
+
+        for (int i = 0; i < padLength; i++) {
+            source += paddingChar;
+        }
+
+        return source;
+    }
+}
+
+/*
```

Comment:
PHP code?

Reply:
do you want php code ?

Reply:
oh :D i was just saving incase someone needs that in php

Reply:
Please remove that.